common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/LocalStatisticsDownload.java

```diff
@@ -0,0 +1,37 @@
+package app.coronawarn.server.common.persistence.domain;
+
+import org.springframework.data.annotation.Id;
+
+public class LocalStatisticsDownload {
```

Comment:
Mismatch between table name and entity. So the repository non-custom methods like findAll() will fail because they look for the wrong table.

Reply:
Hi Felix. The spring repository is generic, thus the entity class is passed to the interface. It is still a good ideea to have the same name but I followed the statistics_dowloaded table name and entity for consistence purposes.

Reply:
I dont understand? If you execute findAll() with the generic repository you created for local statistics then it will fail because it will lookup the wrong table